pymesync/pymesync.py

```diff
@@ -817,6 +817,13 @@ def __construct_filter_query(self, queries):
 
         # Everthing is a list now, so iterate through and append
         else:
+            # Put "start" and "end" queries into lists
```

Comment:
So this list representation is only an internal representation, not given to the outside user? Please make that clear if it's the case.